### ISSUE TITLE
Discuss: location of Coffee Capsules, Cups Etc.

### ISSUE THREAD

**Issue by nelsonic** [dwyl/home#214]
At present the Coffee Cups & Capsules are above the Vitamix in one of the taller cupboards:

![7253EADA-E1EA-47D1-8994-59763A010FF2](https://user-images.githubusercontent.com/194400/207889889-8b7ac296-7c14-4c47-b270-d4922d93e48c.jpeg)

however the cupboard directly above the Nespresso machine is full of _relatively_ infrequently used dry ingredients:

![B0CD4F37-EF30-4A69-8D06-030F9DF73338](https://user-images.githubusercontent.com/194400/207890438-bcf8c7aa-ede5-4c84-a90d-64675ef0ef84.jpeg)

Seems like an inefficient use of space from a organisation + workflow perspective. 💭
We often have to _explain_ to guests where the capsules are … ⏳
Does anyone have any _objection_ to me moving the capsules and other coffee-related items to directly above the Nespresso machine? ☕️

**Comment by iteles** [comment]
No objection to moving those there.

But the pasta section gets used every other day in this house so please leave it just as accessible as it currently is 😬 
If you're doing a focussed batch of computer work in the coming two days during the day, I can focus on a re-org of that kind of thing.

**Comment by nelsonic** [comment]
Thanks for quick reply. Only reason I pinged you about this is that the capsules are currently out on the counter for installing the Kitchen Speakers 🔊 sooo rather than pack things away in their “old” location, was just gonna put them in the _right/logical_ place immediately. It’s definitely priority-3 (“nice to have”) but given that I’m tidying up after mounting the speakers I figured I could get it done immediately. 
if you want to do a wider re-org, that would be awesome. But is moving the capsules + cups, a T5m task _dependent_ on that? 💭
